hack/install-kind.sh

```diff
@@ -15,9 +15,11 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 # kind version
 KIND_VERSION="${KIND_VERSION:-v0.11.1}"
 
-if [ ! -f "${GOPATH}/bin/kind" ] ; then
+if ! hash kind > /dev/null 2>&1 ; then
     echo "# Installing KinD..."
-    go get sigs.k8s.io/kind@${KIND_VERSION}
+    pushd "${HOME}" > /dev/null
+        go get "sigs.k8s.io/kind@${KIND_VERSION}"
+    popd > /dev/null
```

Comment:
With Go 1.16, we can finally make this one even shorter:
```suggestion
    go install "sigs.k8s.io/kind@${KIND_VERSION}"
```

Did a quick check:
```
$ docker run -it --rm golang:1.16 /bin/bash
root@8b6903f0a6b1:/go# KIND_VERSION="${KIND_VERSION:-v0.11.1}"
root@8b6903f0a6b1:/go# go install "sigs.k8s.io/kind@${KIND_VERSION}"
go: downloading sigs.k8s.io/kind v0.11.1
go: downloading github.com/spf13/pflag v1.0.5
go: downloading github.com/spf13/cobra v1.1.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading k8s.io/apimachinery v0.20.2
go: downloading github.com/alessio/shellescape v1.4.1
go: downloading github.com/mattn/go-isatty v0.0.12
go: downloading golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c
go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
go: downloading sigs.k8s.io/yaml v1.2.0
go: downloading github.com/BurntSushi/toml v0.3.1
go: downloading github.com/evanphx/json-patch v4.9.0+incompatible
go: downloading github.com/pelletier/go-toml v1.8.1
go: downloading github.com/evanphx/json-patch/v5 v5.2.0
go: downloading gopkg.in/yaml.v2 v2.2.8
root@8b6903f0a6b1:/go# kind version
kind v0.11.1 go1.16.8 linux/amd64

Reply:
Ty. Done.